### DISCUSSION TITLE
Hset help

### DISCUSSION THREAD

**Discussion by capyun** [redis/go-redis#2084] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/57035288/166147026-83f04874-f385-4dce-aaa9-8a4511b8b412.png)
Why HSet don't have field parameter?

**Comment by baylac** [comment, marked as answer]
because it supports multiple, you can use it in this way
```go
client.HSet(ctx, key, "field1", value1, "field2", value2)
```
